### PR TITLE
OHRM5X-1103 : Configure Performance Trackers

### PR DESCRIPTION
**Pull request by SharukaNP**
OHRM5X-1103 : Configure Performance Trackers
OHRM5X-1104 :My Trackers BE development and localization support in FE

### CODE REVIEW

File: src/client/src/orangehrmPerformancePlugin/components/ReviewersAutoComplete.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            });
+        } else {
+          resolve([]);
+        }
+      });
+    },
+  },
+};
+</script>
+<style scoped>
```
Comment (Super-Chama) [inline src/client/src/orangehrmPerformancePlugin/components/ReviewersAutoComplete.vue:89]: if past emp tag not used, remove this style

File: src/client/src/orangehrmPerformancePlugin/pages/PerformanceTrackers.vue [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+const trackerNormalizer = data => {
+  return data.map(row => {
+    return {
+      id: row.id,
+      tracker: row.trackerName,
+      addDate: row.addedDate,
+      matureDate: row.modifiedDate,
+      empName: `${row.employee?.firstName} ${row.employee?.lastName}
+          ${row.employee?.terminationId ? ' (Past Employee)' : ''}`,
```
Comment (Super-Chama) [inline src/client/src/orangehrmPerformancePlugin/pages/PerformanceTrackers.vue:109]: Past employee translation

**Review by Super-Chama — CHANGES_REQUESTED** [review]

File: src/client/src/orangehrmPerformancePlugin/pages/PerformanceTrackers.vue (hunk 2 above)
Comment (SharukaNP) [reply src/client/src/orangehrmPerformancePlugin/pages/PerformanceTrackers.vue]: Added

**Comment by SharukaNP** [comment]
This PR is closed since the commits in this are included in a new PR as the commits are interrelated

**Pull request closed without merging**